example/apps/example-app/sections/summary-data-sections.js

```diff
@@ -33,5 +33,8 @@ module.exports = {
   ],
   complaintDetails: [
     'complaintDetails'
+  ],
+  countrySelect: [
+    'countrySelect'
   ]
```

Comment:
I think on the summary page this would be better after address (as it is an address type question) or before complaints detail (as it comes before this question in the form)